src/Playwright.NUnit/PlaywrightTest.cs

```diff
@@ -30,7 +30,19 @@ namespace Microsoft.Playwright.NUnit
 {
     public class PlaywrightTest : WorkerAwareTest
     {
-        public static string BrowserName => (Environment.GetEnvironmentVariable("BROWSER") ?? Microsoft.Playwright.BrowserType.Chromium).ToLower();
+        static PlaywrightTest()
+        {
+            var name = (Environment.GetEnvironmentVariable("BROWSER") ?? Microsoft.Playwright.BrowserType.Chromium).ToLower();
+            BrowserName = name switch
+            {
+                Microsoft.Playwright.BrowserType.Chromium => name,
+                Microsoft.Playwright.BrowserType.Firefox => name,
+                Microsoft.Playwright.BrowserType.Webkit => name,
+                _ => Microsoft.Playwright.BrowserType.Chromium
```

Comment:
Why not throw some understandable error instead? I would be surprised when I use `BROWSER=fierfox` but get Chromium because of the typo.

Reply:
That would be a better option, but, at least in Visual Studio, running the test actually fails silently with the exception. I'll look into this a bit more, see if there's a better alternative to block the test.

Reply:
Heh, turns out I can run `Assert` inside the `SetUp` method =)